modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -1303,6 +1305,16 @@ private void processRegularMessage(
                 }
             }
 
+            if (plc == GridIoPolicy.QUERY_POOL) {
+                Message initMsg = msg.message();
+                byte priority = SqlFieldsQuery.MIN_PRIORITY;
```

Comment:
Let's introduce DEFAULT_PRIORITY and set here insted og MINIMUM. I think we should take 5 as the default to have the ability to run any other query with a bigger and lower priority